lib/ui/settings/advanced_settings_screen.dart

```diff
@@ -119,6 +121,32 @@ class _AdvancedSettingsScreenState extends State<AdvancedSettingsScreen> {
                             const SizedBox(
                               height: 24,
                             ),
+                            MenuItemWidget(
+                              captionedTextWidget: CaptionedTextWidget(
+                                title: S.of(context).archiveSharedAlbums,
```

Comment:
There's no entry for this string in the following file? 
```
lib/l10n/intl_en.arb
```

Ideally, lib/generated/l10n.dart should generate the entry based on `intl_en.arb`

Reply:
I have added to lib/generated/l10n.dart

Reply:
Should I add it to intl_en.arb? @ua741

Reply:
Done